test/integration/browser.ts

```diff
@@ -158,7 +158,7 @@ function runsIntegrationSuite(
         // Uncomment to customize the browsers to test when debugging.
         plugins: <any>{
           local: {
-            browsers: ['firefox', 'chrome', /*'safari'*/],
+            // browsers: ['firefox', 'chrome', /*'safari'*/],
```

Comment:
I left this one because I've found it useful more than once when debugging some browser-specific issue. I deleted the others because they were much less useful.